content/version/2/0/code_of_conduct.fr.md

```diff
@@ -0,0 +1,132 @@
+# Charte Code de Conduite Contributeurs
+
+## Notre engagement
+
+En tant que membres, contributeurs et dirigeants, nous nous 
+engageons à rendre la participation dans notre communauté 
+une expérience sans harcelement pour tous, quel que soit l'âge, 
+la physionomie, la présence d'un handicap visible ou invisible, l'ethnie, 
+les caractéristiques sexuelles, l'identité et l'expression de genre, 
+le niveau d'expérience, l'éducation, le statut socio-économique, 
+la nationalité, l'apparence personnelle, la race, la religion, 
+ou l'identité et l'orientation sexuelle.
+
+Nous nous engageons à agir et interagir de manière à contribuer à une communauté 

```

Comment:
```suggestion
Nous nous engageons à agir et interagir de façon à construire à une communauté 

```

Reply:
Very good idea ! We just need to fix a little typo then.
```suggestion
Nous nous engageons à agir et interagir de façon à construire une communauté
```

Reply:
I find the original phrasing closer to the intent of the English version.

Reply:
I'm resolving this conversation since we have find another way to say it without loosing the intent of the English version.